src/go/pt-k8s-debug-collector/dumper/dumper.go

```diff
@@ -78,16 +77,32 @@ func New(location, namespace, resource string) Dumper {
 				"perconaservermongodbrestores",
 				"perconaservermongodbs",
 			)
+		} else if resourceType(resource) == "psql" {
```

Comment:
I'd call it `pg` but it's up to you

Reply:
Good idea, Support team also likes it. So made the change.